xplique/wrappers/pytorch.py

```diff
@@ -0,0 +1,148 @@
+"""
+Module for having a wrapper for PyTorch's model
+"""
+
+import tensorflow as tf
+import numpy as np
+
+try:
```

Comment:
Move the try in the __init__ (see scipy_sampler in gsa). So I can import Torch wrapper without installing torch